doc/install/mne_python.rst

```diff
@@ -129,10 +129,20 @@ GUI). This will install MNE-Python into the "base" conda environment, which
 should be active by default and should already have the necessary dependencies
 (``numpy``, ``scipy``, and ``matplotlib``).
 
+A second option is to install MNE-Python into its own conda environment
+(instead of installing into the "base" environment). This can be done via::
+
+    $ conda create --name new_environment_name --channel conda-forge mne
```

Comment:
Will this environment be immediately available for use? Or do you need to activate it. If so, should we inform the user how to activate it?

I have multiple environments and need to activate the one I want. But maybe if you only have one it's activated automatically?

Reply:
Newly-created environments are not automatically activated. The message shown in the terminal when creating a new environment (here, "foo") is:
```
#
# To activate this environment, use
#
#     $ conda activate foo
#
# To deactivate an active environment, use
#
#     $ conda deactivate
```
This message appears regardless of how many environments one already has.

The only case where this wouldn't be necessary would be a user who installs everything into "base" environment: with a typical conda installation, the base environment will be set to automatically activate when the terminal is opened (though even that is not a guarantee, it's configurable during anaconda install).

Reply:
Yes! I forgot that it spits that message out. I think we should add a line stating users should run `conda activate new_environment_name` as stated in the message. I suggest this because users may either not read the message (unfortunate but common), or they will read the message and then not see the same instructions here and wonder which instructions to follow.

Reply:
Just a suggestion obliviously, I find it hard not to sound bossy in these reviews.